pkg/process/debug.go

```diff
@@ -82,11 +82,18 @@ func sanitize(val string) string {
 
 func prometheus(w http.ResponseWriter, r *http.Request) {
 	// writes https://prometheus.io/docs/instrumenting/exposition_formats/
-	// TODO(jt): deeper monkit integration so we can expose prometheus types
 	// (https://prometheus.io/docs/concepts/metric_types/)
-	monkit.Default.Stats(func(name string, val float64) {
-		metric := sanitize(name)
-		_, _ = fmt.Fprintf(w, "# TYPE %s gauge\n%s %g\n",
-			metric, metric, val)
+	monkit.Default.Stats(func(series monkit.Series, val float64) {
+		measurement := sanitize(series.Measurement)
+		var metrics []string
+		for tag, tagVal := range series.Tags.All() {
```

Comment:
do we care about the tags being sorted? doing this we get different orderings on the tags every time because of randomized map iteration.

Reply:
:man_facepalming: there was a git-fail at some point a while back that I've been suffering from ever since and it's causing some changes to keep getting reverted.

I agree that we need to coordinate the release, maybe we should just schedule a meeting to do that with the appropriate parties.